doc/source/main_install.rst

```diff
@@ -258,40 +219,23 @@ and set up other necessary environment variables::
 If you do not want to manage your virtual environments manually, you can
 consider using `pipx <https://docs.python.org/3/tutorial/venv.html>`_.
 
+.. _install-completions:
 
-.. _post-install:
-
-Post-install setup
-------------------
-
-After having installed from source using any of the above methods, some
-setup will be required to use BuildStream.
-
-
-
-Adjust ``PATH``
-~~~~~~~~~~~~~~~
-Since BuildStream is now installed under your local user's install directories,
-you need to ensure that ``PATH`` is adjusted.
-
-A regular way to do this is to add the following line to the end of your ``~/.bashrc``::
-
-  export PATH="${PATH}:${HOME}/.local/bin"
-
-.. note::
+Installing completions
+----------------------
 
-   You will have to restart your terminal in order for these changes to take effect.
+BuildStream integrates with Bash to provide helpful tab-completion. These
+are provided by the script ``src/buildstream/data/bst`` which requires
```

Comment:
Wouldn't it be good to expand the content of `src/buildstream/data/bst` here (with `::literalinclude`) or at least provide a link to the master branch version of the `bst` completions script on github ?

Also I thought there was an easier way to get this working for a single user, by simply copying the content of the script (from this page) and pasting the content directly into `~/.bash_completion`.

Reply:
We expect that the user has a Git clone if they're reading these instructions.

I'm not convinced that 'add this text to this file' is much simpler to do than 'copy this file to this location'. It seems simpler to express it though :)

I will rework as described